### PR TITLE
frostenberg docs suggestions

### PR DESCRIPTION
**Pull request by haakon-e**
## Purpose 

follow-up from #328 

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by haakon-e** [comment]
made some updates to the plot, including reversed time axis on the bottom plot :P
image

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,6 @@ docs/site/
 
 # Deps
 Manifest.toml
+
+# Settings
+.vscode/
```
Comment (trontrytel) [inline .gitignore:28]: 😲

File: docs/src/IceNucleationParcel0D.md [hunk 2]
```diff
@@ -353,46 +353,47 @@ Three different implementations of this parametrization are used in the parcel m
...
   The inverse timescale of the process is set by ``\gamma``.
 
-The stochastic implementation is based on the equation for a generic
-  stochastic process ``x`` with Gaussian random noise (i.e. Wiener process):
+The stochastic implementation is based on the [Ornstein-Uhlenbeck process](https://en.wikipedia.org/wiki/Ornstein–Uhlenbeck_process), 
+in which the variable ``x`` is a mean-reverting process perturbed by Gaussian random noise (i.e. Wiener process):
 ```math
 \begin{equation}
-  dx = - \gamma \; x \; dt + g dW; \;\;\;\;\;\;\;   dW = {\bf{N}}(0, dt^2),
+  dx = - \gamma(x - \mu)dt + \sqrt{2\gamma} \sigma dW; \quad\quad   dW \sim \mathcal{N}(0, dt),
```
Comment (trontrytel) [inline docs/src/IceNucleationParcel0D.md:360]: `mathcal` doesn't render properly. I think it's ok to stick with `bf`

Comment (trontrytel) [reply docs/src/IceNucleationParcel0D.md:360]: Also, what is the justification for multiplying the second term by `\sqrt{2\gamma}` ? Thats not how the problem is formulated in the wiki link you provided?

File: docs/src/IceNucleationParcel0D.md [hunk 3]
```diff
@@ -353,46 +353,47 @@ Three different implementations of this parametrization are used in the parcel m
...
-  `random` (dotted lines) and `stochastic` (solid lines) parameterization options.
-We show results for two sampling intervals ``\Delta t`` (random),
-  two process time scales ``\tau`` (stochastic),
-  and two model time steps `dt`.
+The following plot shows resuls of the parcel model with the `mean` (black line), `random` (dotted lines) and `stochastic` (solid lines) parameterization options.
+We show results for two sampling intervals ``\Delta t`` (random), two process time scales ``\tau`` (stochastic), and two model time steps `dt`.
 
 ```@example
 include("../../parcel/Example_Frostenberg_Immersion_Freezing.jl")
+nothing # hide
```
Comment (trontrytel) [inline docs/src/IceNucleationParcel0D.md:397]:
```suggestion
```

File: parcel/Example_Frostenberg_Immersion_Freezing.jl [hunk 4]
```diff
@@ -138,18 +138,20 @@ end
...
-ax4 = MK.Axis(fig[2, 2], ylabel = "q_liq [g/kg]")
-ax5 = MK.Axis(fig[3, 1], xlabel = "Time [min]", ylabel = "N_ice [1/cm3]")
-ax6 = MK.Axis(fig[3, 2], xlabel = "Time [min]", ylabel = "N_liq [1/cm3]")
-ax7 = MK.Axis(
-    fig[4, 1:2],
-    xlabel = "T [K]",
-    ylabel = "INPC [1/m3]",
-    yscale = log10,
-)
+kwargs = (; xgridvisible = false, ygridvisible = false,)
```
Comment (trontrytel) [inline parcel/Example_Frostenberg_Immersion_Freezing.jl:141]: Not sure I'm a fan of the temperature going backwards in time. But if you think it's better, lets keep it :)

**Review by trontrytel — APPROVED** [review]
Thank you!

**Pull request merged**